src/main/cms/cms_menu_osd.c

```diff
@@ -118,6 +118,7 @@ const OSD_Entry menuOsdActiveElemsEntries[] =
     {"PID PROFILE NAME",   OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_PID_PROFILE_NAME], DYNAMIC},
     {"RATE PROFILE NAME",  OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_RATE_PROFILE_NAME], DYNAMIC},
 #endif
+    {"OSD PROFILE NAME",   OME_VISIBLE, NULL, &osdConfig_item_pos[OSD_PROFILE_NAME], DYNAMIC},
```

Comment:
Should be bounded with `#ifdef USE_OSD_PROFILES`

Reply:
Done.